src/renderer.ts

```diff
@@ -65,19 +70,51 @@ export class Renderer {
       const page = await newIncognitoBrowserContext.newPage();
       await page.setRequestInterception(true);
 
-    page.on('request', (interceptedRequest) => {
-      const interceptedUrl = interceptedRequest.url();
-      const allowedUrlsRegex = /^https?:\/\/(.*?).?gozefo.com.*/;
-      // console.log('interceptedUrl: ', interceptedUrl, 'allowed: ', interceptedUrl.match(allowedUrlsRegex) ? 'true' : false);
-      if (!interceptedUrl.match(allowedUrlsRegex))
-        interceptedRequest.abort();
-      else
-        interceptedRequest.continue();
-    });
+      page.on('request', (interceptedRequest: Request) => {
+        const interceptedUrl = interceptedRequest.url();
+        // const allowedUrlsRegex = /^https?:\/\/(.*?).?gozefo.com.*/;
+        // console.log('interceptedUrl: ', interceptedUrl, 'allowed: ', interceptedUrl.match(allowedUrlsRegex) ? 'true' : false);
+        if (!interceptedUrl.match(Renderer.ALLOWED_URL_PATTERN))
+          interceptedRequest.abort();
+        else if(interceptedUrl.match(Renderer.CACHE_URL_PATTERN)) {
+          if (((new Date()).getTime() - this.responseCacheStartTimeStamp) > Renderer.CACHE_EXPIRY) {
```

Comment:
I think the cache may be at key/file level rather than expiring the entire cache